Give DAHDI PtMP NT mode a shot.

This basically sort-of works. But then again, implementing the whole mode
in a three-liner patch is quite a feat ;-)

http://bugs.digium.com/view.php?id=15048
--- a/channels/chan_dahdi.c
+++ b/channels/chan_dahdi.c
@@ -14582,7 +14582,8 @@ static int process_dahdi(struct dahdi_ch
 					confp->chan.sig = SIG_BRI_PTMP;
 					confp->pri.nodetype = PRI_CPE;
 				} else if (!strcasecmp(v->value, "bri_net_ptmp")) {
-					ast_log(LOG_WARNING, "How cool would it be if someone implemented this mode!  For now, sucks for you. (line %d)\n", v->lineno);
+					confp->chan.sig = SIG_BRI_PTMP;
+					confp->pri.nodetype = PRI_NETWORK;
 				} else if (!strcasecmp(v->value, "gr303fxoks_net")) {
 					confp->chan.sig = SIG_GR303FXOKS;
 					confp->pri.nodetype = PRI_NETWORK;
